codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -167,6 +165,10 @@ public Http2ConnectionEncoder encoder() {
         return encoder;
     }
 
+    public void frameListener(Http2FrameListener listener) {
```

Comment:
Yah I'm on the fence. From my standpoint I think its easy, but I could see how users who just create a `new Http2ConnectionHandler()` may not want to know about implementation details to have to get the decoder and set the listener.  I guess we could remove it for now...and if it becomes a problem add it later?